app/controllers/reportings_controller.rb

```diff
@@ -197,18 +199,7 @@ def new
   end
 
   def create
-    @reporting = @project.reportings_via_source.build
-    @reporting.reporting_to_project_id = params['reporting']['reporting_to_project_id']
-
-    if @reporting.reporting_to_project.nil?
-      flash.now[:error] = l('timelines.reporting_could_not_be_saved')
-      render action: :new, status: :unprocessable_entity
```

Comment:
Erm,

the 422 was actually introduced by me in the hotfix to indicate a failed request - it had not been there beforehand,a s the standard response in an error case was either just to render new or to respond with 404.